ci-operator/step-registry/nutanix/provision/bastionhost/nutanix-provision-bastionhost-ref.yaml

```diff
@@ -0,0 +1,29 @@
+ref:
+  as: nutanix-provision-bastionhost
+  from: upi-installer
+  cli: latest
+  grace_period: 10m
+  commands: nutanix-provision-bastionhost-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: BASTION_SUBNET_UUID
+    default: "ae6e2fd8-79fe-4a88-a0d0-7d66cc45bdb1"
```

Comment:
The default subnet is the disconnected segment network?

Reply:
No, the bastion host is in connected subnet where disconnect network can access too.

Reply:
Got it.